src/org/labkey/test/tests/wiki/WikiAliasesTest.java

```diff
@@ -0,0 +1,113 @@
+    @BeforeClass
+    public static void setupProject()
+    {
+        WikiAliasesTest init = (WikiAliasesTest) getCurrentTest();
+        init.doSetup();
+    }
+
+    @Override
+    protected @Nullable String getProjectName()
+    {
+        return getClass().getSimpleName() + " Project";
+    }
+
+    private void doSetup()
+    {
+        _containerHelper.createProject(getProjectName(), null);
+        _containerHelper.enableModules(Arrays.asList("Wiki"));
+
+        _containerHelper.createSubfolder(getProjectName(), SUBFOLDER);
+        _containerHelper.enableModules(Arrays.asList("Wiki"));
+
+        SearchAdminAPIHelper.pauseCrawler(getDriver());
+
+        goToProjectHome();
+        PortalHelper portalHelper = new PortalHelper(this);
+        portalHelper.addBodyWebPart("Wiki");
+
+        navigateToFolder(getProjectName(), SUBFOLDER);
+        portalHelper.addBodyWebPart("Wiki");
+    }
+
+    @Test
+    public void testSteps()
+    {
+        goToProjectHome();
+        WikiHelper wikiHelper = new WikiHelper(this);
+        String duplicateAlias = "Alias5";
+        String aliases = "Alias1\n" + "Alias2\n" + "Alias3\n" + "Alias4\n" + duplicateAlias;
+        wikiHelper.createWikiPage(wikiName, null, wikiTitle, wikiBody, true, null, false);
+
+        log("Verifying adding new alias with rename button");
+        ManageWikiConfigurationPage manageWikiConfigurationPage = wikiHelper.manageWikiConfiguration();
+        manageWikiConfigurationPage.rename(duplicateAlias, true)
+                .setAliases(aliases)
+                .save();
+
+        log("Verifying  all the aliases added works with wiki-page.view");
+        manageWikiConfigurationPage = wikiHelper.manageWikiConfiguration();
+        String listOfAliases[] = manageWikiConfigurationPage.getAliases().split("\n");
+        for (int i = 0; i < listOfAliases.length; i++)
+        {
+            beginAt(WebTestHelper.buildURL("wiki", getProjectName(), "page", Map.of("name", listOfAliases[i])));
+            Assert.assertEquals("Incorrect wiki body for alias " + listOfAliases[i], wikiBody,
+                    Locator.tagWithClass("div", "labkey-wiki").findElement(getDriver()).getText());
+        }
+
+        log("Verifying same alias name is not allowed");
+        manageWikiConfigurationPage = wikiHelper.manageWikiConfiguration();
+        String errMsg = manageWikiConfigurationPage.setAliases( "\n" + duplicateAlias)
+                .saveExpectingErrors();
+        Assert.assertEquals("Incorrect error message", "Warning: Alias '" + duplicateAlias + "' already exists in this folder.",errMsg);
+
+        log("Testing case insensitive");
+        manageWikiConfigurationPage = wikiHelper.manageWikiConfiguration();
+        errMsg = manageWikiConfigurationPage.setAliases("\n" + duplicateAlias.toLowerCase()).saveExpectingErrors();
+        Assert.assertEquals("Incorrect error message", "Warning: Alias '" + duplicateAlias.toLowerCase() + "' already exists in this folder.",errMsg);
+
```

Comment:
A good additional test would be to create a new wiki with a name that matches one of the aliases (you could use `wikiName`) and verify that page.view resolves to the wiki with that name, not the wiki that has that alias.

Reply:
Good suggestion